PokeOneWeb.DataSync/ReadModelUpdate/Impl/ReadModelUpdateService.cs

```diff
@@ -100,7 +104,127 @@ public void UpdateReadModel(SpreadsheetImportReport importReport)
             _regionRepository.Update(_regionMapper.MapFromDatabase(importReport));
             _reporter.StopReadModelUpdate("regions");
 
+            _reporter.StartReadModelUpdate("generate-json-files");
+            GenerateJsonFiles(importReport);
+            _reporter.StopReadModelUpdate("generate-json-files");
+
             _reporter.StopReadModelUpdate();
         }
+
+        private void GenerateJsonFiles(SpreadsheetImportReport importReport)
+        {
+            CreateDirectories();
+
+            var serializeOptions = new JsonSerializerOptions
+            {
+                PropertyNamingPolicy = JsonNamingPolicy.CamelCase,
+                WriteIndented = true
+            };
+
+            //
+            // Entities
+            // 
+            Console.WriteLine("generating json files for entities");
+            ICollection<EntityTypeReadModel> entities = _entityTypeMapper.MapFromDatabase(importReport).Keys;
+            File.WriteAllText("resources/entities.json", JsonSerializer.Serialize(entities, serializeOptions));
```

Comment:
Nomenclature: Name this type of ressource consistently "EntityTypes", not "Entities". This resource resolves for every resource name, what *type* of entity that resource is.

Reply:
👍